tox.ini

```diff
@@ -13,11 +13,12 @@ extras =
 deps =
     pipprevious: pip==21.3.*
     piplatest: pip
+    piplatest: -e git+https://github.com/pypa/pip.git@main#egg=pip
     pipmain: -e git+https://github.com/pypa/pip.git@main#egg=pip
 setenv =
     coverage: PYTEST_ADDOPTS=--strict --doctest-modules --cov --cov-report=term-missing --cov-report=xml {env:PYTEST_ADDOPTS:}
 commands_pre =
-    piplatest: python -m pip install -U pip
+    piplatest: python -m pip install -U pip --ignore-installed
```

Comment:
```suggestion
    piplatest: python -m pip install "pip @ git+https://github.com/pypa/pip.git"
```

Reply:
Hey! Thanks for chiming in ^___^ I've reverted my test changes, so this is outdated now.